config.json

```diff
@@ -1,5 +1,6 @@
 {
   "language": "D",
+  "blurb": "D is a general-purpose programming language with static typing, systems-level access, and C-like syntax. With the D Programming Language, write fast, read fast, and run fast.",
```

Comment:
I think this is the language's official introduction text verbatim, right? We usually don't want to copy things literally due to licensing issues. In https://github.com/exercism/d/pull/144/files, I took a stab at writing a blurb. How does that look like? Maybe you can use that and tweak it a bit?

Reply:
I opted to leave your blurb as-is. I think it fits the language just fine!